sympy/logic/tests/test_boolalg.py

```diff
@@ -708,3 +710,65 @@ def test_truth_table():
     assert list(truth_table(And(x, y), [x, y], input=False)) == [False, False, False, True]
     assert list(truth_table(x | y, [x, y], input=False)) == [False, True, True, True]
     assert list(truth_table('x >> y', ['x', 'y'], input=False)) == [True, True, False, True]
+
+
+def test_to_anf():
+    x, y = symbols('x,y')
+    assert to_anf(And(x, y)) == And(x, y)
+    assert to_anf(Or(x, y)) == Xor(x, y, And(x, y))
+    assert to_anf(Xor(x, y)) == Xor(x, y)
+    assert to_anf(Not(x)) == Xor(x, True, remove_true=False)
+    assert to_anf(Nand(x, y)) == Xor(True, And(x, y), remove_true=False)
+    assert to_anf(Nor(x, y)) == Xor(x, y, True, And(x, y), remove_true=False)
+    assert to_anf(Implies(x, y)) == Xor(x, True, And(x, y), remove_true=False)
+    assert to_anf(Equivalent(x, y)) == Xor(x, y, True, remove_true=False)
+    assert to_anf(Nand(x | y, x >> y), deep=False) == \
+           Xor(True, And(Or(x, y), Implies(x, y)), remove_true=False)
+    assert to_anf(Nor(x ^ y, x & y), deep=False) == \
+           Xor(True, Or(Xor(x, y), And(x, y)), remove_true=False)
+
+
+def test_is_anf():
+    x, y = symbols('x,y')
+    assert is_anf(true) is True
+    assert is_anf(false) is True
+    assert is_anf(x) is True
+    assert is_anf(And(x, y)) is True
+    assert is_anf(Xor(x, y, And(x, y))) is True
+    assert is_anf(Xor(x, y, Or(x, y))) is False
+    assert is_anf(Xor(Not(x), y)) is False
+
+
+def test_anf_coeffs():
+    assert anf_coeffs([1, 0]) == [1, 1]
+    assert anf_coeffs([0, 0, 0, 1]) == [0, 0, 0, 1]
+    assert anf_coeffs([0, 1, 1, 1]) == [0, 1, 1, 1]
+    assert anf_coeffs([1, 1, 1, 0]) == [1, 0, 0, 1]
+    assert anf_coeffs([1, 0, 0, 0]) == [1, 1, 1, 1]
+    assert anf_coeffs([1, 0, 0, 1]) == [1, 1, 1, 0]
+    assert anf_coeffs([1, 1, 0, 1]) == [1, 0, 1, 1]
+
+
+def test_ANF():
+    x, y = symbols('x,y')
+    assert ANF([x], [1, 1]) == True
+    assert ANF([x], [0, 0]) == False
+    expr = ANF([x], [1, 0])
+    assert isinstance(expr, Xor) and expr.args == (x, True)
+    expr = ANF([x, y], [1, 1, 1, 0])
+    assert isinstance(expr, Xor) and expr.args == (True, And(x, y))
+
+def test_minterm():
+    x, y = symbols('x,y')
+    assert minterm(3, ['x', 'y']) == And(x, y)
+    assert minterm([1, 0], [x, y]) == And(Not(y), x)
+
+def test_maxterm():
+    x, y = symbols('x,y')
+    assert maxterm(2, ['x', 'y']) == Or(Not(x), y)
```

Comment:
Don't use strings in the tests either. Strings as input is something that should only work incidentally. See https://github.com/sympy/sympy/wiki/Idioms-and-Antipatterns#strings-as-input.